lms/services/assignment.py

```diff
@@ -204,9 +204,11 @@ def upsert_assignment_groupings(
     def get_by_id(self, id_: int) -> Assignment | None:
         return self._db.query(Assignment).filter_by(id=id_).one_or_none()
 
-    def is_member(self, assignment: Assignment, user: User) -> bool:
+    def is_member(self, assignment: Assignment, h_userid: str) -> bool:
```

Comment:
Same pattern as in courses recently. Missed we did the wrong thing for assignments when doing that change.